task3.java

```diff
@@ -0,0 +1,36 @@
+package com.Selenium.homework1;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class task3 {
+
+
+
+    public static void main(String[] args) {
+        Integer[] masyv = { 3, 43,7,7,0, 54, 3,1, 2,0, 6, 43,0,1,0, 2,};
+        Integer min = masyv[0];
+        Integer average = 0;

```

Comment:
do we need to use Integer object? why not just int?

Reply:
fixed in 'task3.changes'